### PR TITLE
Update account discount and tax issue resolbved

### PR DESCRIPTION
**Pull request by rohit-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/ledger/components/newLedgerEntryPanel/newLedgerEntryPanel.component.ts [hunk 1]
```diff
@@ -163,8 +163,9 @@ export class NewLedgerEntryPanelComponent implements OnInit, OnDestroy, OnChange
     public mapBodyContent: string;
     public selectedItemToMap: ReconcileResponse;
     public tags$: Observable<TagRequest[]>;
-    public activeAccount$: Observable<AccountResponse>;
-    public activeAccount: AccountResponse;
+    public activeAccount$: Observable<AccountResponse | AccountResponseV2>;
+    public activeUpdatedAccount$: Observable<AccountResponse | AccountResponseV2>;
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/ledger/components/newLedgerEntryPanel/newLedgerEntryPanel.component.ts:167]: Add JS doc

File: apps/web-giddh/src/app/ledger/components/newLedgerEntryPanel/newLedgerEntryPanel.component.ts [hunk 2]
```diff
@@ -243,6 +244,7 @@ export class NewLedgerEntryPanelComponent implements OnInit, OnDestroy, OnChange
         this.sessionKey$ = this.store.select(p => p.session.user.session.id).pipe(takeUntil(this.destroyed$));
         this.companyName$ = this.store.select(p => p.session.companyUniqueName).pipe(takeUntil(this.destroyed$));
         this.activeAccount$ = this.store.select(p => p.ledger.account).pipe(takeUntil(this.destroyed$));
+        this.activeUpdatedAccount$ = this.store.select(p => p.ledger.account).pipe(takeUntil(this.destroyed$));
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/ledger/components/newLedgerEntryPanel/newLedgerEntryPanel.component.ts:247]: Avoid single character variable

Comment (ravinderwalkover) [inline apps/web-giddh/src/app/ledger/components/newLedgerEntryPanel/newLedgerEntryPanel.component.ts:247]: update the syntax and don't use single character var

File: apps/web-giddh/src/app/ledger/components/newLedgerEntryPanel/newLedgerEntryPanel.component.ts [hunk 3]
```diff
@@ -1408,4 +1394,35 @@ export class NewLedgerEntryPanelComponent implements OnInit, OnDestroy, OnChange
...
         this.adjustPaymentModal.show();
     }
+
+    /**
+     * To assign updated account details
+     *
+     * @param {(AccountResponse | AccountResponseV2)} accountDetails
+     * @memberof NewLedgerEntryPanelComponent
+     */
+    public assignUpdateActiveAccount(accountDetails: AccountResponse | AccountResponseV2) {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/ledger/components/newLedgerEntryPanel/newLedgerEntryPanel.component.ts:1404]: Add return type

**Review by prateek-giddh — CHANGES_REQUESTED** [review]

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

**Pull request closed without merging**